packages/uiweb/src/lib/components/chat/ChatPreview/ChatPreview.tsx

```diff
@@ -41,7 +41,7 @@ export const ChatPreview: React.FC<IChatPreviewProps> = (options: IChatPreviewPr
       setFormattedAddress(address);
       if (!options.chatPreviewPayload?.chatGroup) {
         try {
-          const result = await resolveWeb3Name(address, user);
+          const result = await resolveWeb3Name(address, user?.env || CONSTANTS.ENV.PROD);
```

Comment:
Instead of applying this conditional check at the time of using the function let's apply this check inside  resolveWeb3Name function